security-advisories/mbedtls-security-advisory-2024-01-2.md

```diff
@@ -0,0 +1,31 @@
+# Buffer overflow in mbedtls_x509_set_extension()
+
+**Title** |  Buffer overflow in mbedtls_x509_set_extension().
+---|---
+**CVE** |  CVE-2024-23775
+**Date** |  9th of January, 2024
+**Affects** |  All versions 2.16.1 / 2.7.10, fixed in 3.52 / 2.28.7
```

Comment:
What does "All versions 2.16.1 / 2.7.10" mean? "All versions since 2.7.10"?

Reply:
And fixed in `3.5.2` not `3.52`

Reply:
Yes, and yes